cmd/validate/__snapshots__/image_test.snap

```diff
@@ -13,20 +13,20 @@
  "effective-time": "1970-01-01T00:00:00Z",
  "key": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAECBtqKHcvxYkGx7ZXqps3nrYS+ZSA\nmh3m1MZfTGlnr2oN0z+sBWEC23s4RkVSXkEydI6SLYatUtJK8OmiBRS+Xw==\n-----END PUBLIC KEY-----\n",
  "policy": {
-  "configuration": {
-   "exclude": [
-    "not_useful",
-    "test:conftest-clair"
-   ],
-   "include": [
-    "always_checked",
-    "@salsa_one_collection"
-   ]
-  },
   "description": "My custom enterprise contract policy configuration",
   "publicKey": "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAECBtqKHcvxYkGx7ZXqps3nrYS+ZSA\nmh3m1MZfTGlnr2oN0z+sBWEC23s4RkVSXkEydI6SLYatUtJK8OmiBRS+Xw==\n-----END PUBLIC KEY-----\n",
   "sources": [
    {
+    "config": {
+     "exclude": [
+      "not_useful",
+      "test:conftest-clair"
+     ],
+     "include": [
+      "always_checked",
+      "@salsa_one_collection"
```

Comment:
Out of scope, but I wondered - is this an intentional or unintentional comedy misspelling?

Reply:
It looks like @zregvart added it in https://github.com/enterprise-contract/ec-cli/pull/1076 probably for the lolz.